### DISCUSSION TITLE
Warning: read-pkg-up deprecated

### DISCUSSION THREAD

**Discussion by MrGadget1024** [semantic-release/semantic-release#3352] [Release workflow]
Where does this need to be fixed?

![image](https://github.com/semantic-release/semantic-release/assets/9826063/d113aa00-d532-4f9e-bf0a-5d60e893e445)

![image](https://github.com/semantic-release/semantic-release/assets/9826063/1dd5ac4a-6358-4d43-9b42-f59484a74291)

**Comment by gr2m** [comment]
using `npm ls` is a great way to find that out. 

```
➜  semantic-release git:(master) ✗ npm ls read-pkg-up
semantic-release@0.0.0-development /Users/gr2m/code/semantic-release/semantic-release
└─┬ @semantic-release/release-notes-generator@14.0.0
  └── read-pkg-up@11.0.0
```

Looks like it needs to be fixed in https://github.com/semantic-release/release-notes-generator. PR welcome!

**Comment by MrGadget1024** [comment]
PR https://github.com/semantic-release/release-notes-generator/pull/671
